appinventor/appengine/war/tutorials/Tutorial_DigitalDoodle.js

```diff
@@ -0,0 +1,117 @@
+var Tutorial_DigitalDoodle = {
+    title: "Digital Doodle",
+    difficulty: "easy",
+    steps: [
+        {
+            text: 'This tutorial will show you how to draw a line on the screen as the user drags a finger around.',
+            validate: function (formName) {
+                return true;
+            },
+            url: '',
+        },
+        {
+            text: 'To allow the Canvas to fill the whole screen, verify that the Screen1 property "Scrollable" is unchecked (the box is near the bottom of the Properties pane).',
+            validate: function (formName) {
+//                return Tutorial.testForScreenProperty({ Scrollable: false});
```

Comment:
The indentation in the file is consistent, I think. Should the indents be smaller? Right now they are 4 spaces.